.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/elixir-starter-kit
+    docker:
+      - image: circleci/elixir:1.6
+        environment:
+          - MIX_ENV=test
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-deps-{{ .Branch }}-{{ checksum "mix.lock" }}
+            - v1-deps-{{ .Branch }}
+            - v1-deps
+      - restore_cache:
+          keys:
+            - v1-build-{{ .Branch }}
+            - v1-build
```

Comment:
Does this restore the cahe for the `_build` directory? I agree with caching the deps build, but I do find value in having the tests compile from a clean state on ever test run. What are you thoughts?